app/controllers/product_drives_controller.rb

```diff
@@ -6,10 +6,13 @@ def index
     setup_date_range_picker
     @product_drives = current_organization
                      .product_drives
+                     .includes(donations: [line_items: [:item]])
```

Comment:
This is a weird syntax... I'd assume it should look like `donations: {line_items: :item}`. Not even sure how this parses :open\_mouth:

Reply:
Functionally they create the same SQL query but describe the associations using arrays vs using a hash as you have. 

I've changed it as you suggested as it probably is better to have it described as a hash and it a bit more readable.